docs/GLOSSARY.md

```diff
@@ -145,19 +179,27 @@ Commonly used to refer to a [Layout](#layout).
 
 Internally, this is used only to refer to the value used to initialize a [Random Number Generator](#rng).
 
+## Session
+
+An instance of [Multiworld](#multiworld) play.
```

Comment:
What are the bounds of a session (when do they start/stop)?

Reply:
I'm actually not super clear on that myself, my apologies. I don't really play multiworld.